Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -2078,33 +2079,36 @@ public Expression resolveIdentifier(String identifier, boolean mustResolve) {
         // 10: The name of a property on a specific context
         // 11: An unresolved identifier error is thrown
 
+        //Collect every match, issue warning if multiple matches were made, but return initial match:
+        List<Expression> matchCollection = new ArrayList<>();
+
         // In a type specifier context, return the identifier as a Literal for resolution as a type by the caller
         if (inTypeSpecifierContext()) {
-            return this.createLiteral(identifier);
+            matchCollection.add(this.createLiteral(identifier));
         }
 
         // In the sort clause of a plural query, names may be resolved based on the result type of the query
         IdentifierRef resultElement = resolveQueryResultElement(identifier);
         if (resultElement != null) {
-            return resultElement;
+            matchCollection.add(resultElement);
         }
 
         // In the case of a $this alias, names may be resolved as implicit property references
         Expression thisElement = resolveQueryThisElement(identifier);
         if (thisElement != null) {
-            return thisElement;
+            matchCollection.add(thisElement);
```

Comment:
String.format("%s resolved as an element of the current iteration element", identifier)